examples/ts/proxy/create-wallet.ts

```diff
@@ -0,0 +1,35 @@
+import { BitGo } from 'bitgo';
```

Comment:
New examples should incorporate modularization methods. See: https://github.com/BitGo/BitGoJS/blob/master/examples/ts/get-wallet.ts